src/xradio/vis/_vis_utils/_ms/msv2_to_msv4_meta.py

```diff
@@ -1,6 +1,7 @@
 col_to_data_variable_names = {
     "FLOAT_DATA": "SPECTRUM",
     "DATA": "VISIBILITY",
+    "MODEL_DATA": "VIS_MODEL",
```

Comment:
"VIS_MODEL" and "VISIBILITY_CORRECTED" is inconsistent, should both be "VIS_" or "VISIBILITY_"